schemas/config-schema.json

```diff
@@ -215,9 +215,18 @@
                                 "default": "goodtables",
                                 "examples": [
                                     "goodtables"
+                                ],
+                            "expect-header": {
+                                "$id": "#/properties/tables/items/properties/table1/properties/expect-header",
+                                "type": "boolean",
+                                "title": "The expect-header Schema",
+                                "description": "If the file is a csv, does it have headers? If true will take header names from the meta data",
```

Comment:
metadata should be one word

Reply:
haha of course, are you sure it isn't `meta_data` though?

Reply:
(I will change to metadata)